src/coreclr/tools/aot/DependencyGraphViewer/Program.cs

```diff
@@ -143,16 +146,125 @@ struct GraphEvent
         public int Num3;
         public string Str;
     }
+    class DGMLGraphProcessing
+    {
+        ConcurrentQueue<GraphEvent> events = new ConcurrentQueue<GraphEvent>();
+
+        public delegate void OnCompleted(int currFileID);
+        public event OnCompleted Complete;
+
+        public DGMLGraphProcessing(int file)
+        {
+            FileID = file;
+        }
+        public int FileID
+        {
+            get; init;
+        }
+
+        public void FindXML(string argPath)
+        {
+            var fileContent = string.Empty;
+            var filePath = string.Empty;
+            var fileStream = System.IO.Stream.Null;
```

Comment:
Nit: Since you declare that you are using System.IO these names can be simplified for Stream.Null. Same for other parts of this file.

Reply:
thanks! replaced